common/types.d.ts

```diff
@@ -194,6 +201,12 @@ declare global {
   type ILookup = { [key: string]: any }
 
   type IPreferences = ILookup
+
+  type IManufacturer = {
```

Comment:
```
interface ManufacturerInfo {
```

Reply:
Do you want this to be IManufacturerInfo?

Reply:
He is saying that new thinking is to not Prepend the `I` but everything is already with the `I` so maybe at some point we can convert everything. I'm not sure it's super high priority.